security/clamav/src/opnsense/mvc/app/models/OPNsense/ClamAV/Url.xml

```diff
@@ -0,0 +1,21 @@
+<model>
+    <mount>//OPNsense/clamav/url</mount>
+    <description>ClamAV List configuration</description>
+    <version>0.0.1</version>
+    <items>
+        <lists>
+            <list type="ArrayField">
+                <enabled type="BooleanField">
+                    <default>1</default>
+                    <Required>Y</Required>
+                </enabled>
+                <name type="TextField">
+                    <Required>Y</Required>
+                </name>
+                <link type="TextField">
+                    <Required>Y</Required>
```

Comment:
should have a mask like starts with http at least

Reply:
good idea! shouldn't this work? `<mask>/^https?:\/\/*$/i</mask>`

Reply:
Add a dot before the asterisk. Then it means any charachter 0 to infinity times.

Reply:
Thx, fixed